workflows/src/main/java/net/corda/solarsystem/flows/LaunchProbeFlowAcceptorJava.java

```diff
@@ -0,0 +1,43 @@
+package net.corda.solarsystem.flows;
+
+import net.corda.systemflows.ReceiveFinalityFlow;
+import net.corda.systemflows.SignTransactionFlow;
+import net.corda.v5.application.flows.Flow;
+import net.corda.v5.application.flows.FlowSession;
+import net.corda.v5.application.flows.InitiatedBy;
+import net.corda.v5.application.flows.flowservices.FlowEngine;
+import net.corda.v5.application.injection.CordaInject;
+import net.corda.v5.base.annotations.Suspendable;
+import net.corda.v5.ledger.transactions.SignedTransaction;
+import org.jetbrains.annotations.NotNull;
+
+@InitiatedBy(LaunchProbeFlowJava.class)
+public class LaunchProbeFlowAcceptorJava implements Flow<SignedTransaction> {
```

Comment:
When this class was package-private declared inside another file, I got the "LaunchProbeFlowJava not registered"... note that it didn't say "LaunchProbeFlowAcceptorJava not registered". So anyway, it was a little hard to track down, moving to its own file and making it public seemed to fix it.